src/features/appearance/index.ts

```diff
@@ -20,6 +21,15 @@ function createStyleElement() {
   document.querySelector('head')?.appendChild(styles);
 }
 
+function injectUserCustomCSS() {
+  const contentToInject = loadCustomCSS();
```

Comment:
Is it better/more maintainable to inline `loadCustomCSS` in this file/line rather than keeping it in a separate file?

Reply:
I agree, moving it now